CRM/Utils/Number.php

```diff
@@ -88,6 +88,8 @@ public static function createTruncatedDecimal($keyValue, $precision) {
   public static function formatUnitSize($size, $checkForPostMax = FALSE) {
     if ($size) {
       $last = strtolower($size{strlen($size) - 1});
+      $size = (int) $size;
+      
```

Comment:
@Jazz-Man there appears to be whitespace on this line that is causing issues with out test bot. Are you able to fix it?

Reply:
yes, I'll try right now